aineko/extras/fastapi/health.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Health endpoint for FastAPI server."""
+
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get("/health", status_code=200)
+async def read_health() -> dict:
+    """Return the health status of the Aineko FastAPI server.
+
+    Health router can be imported by an Aineko FastAPI server and added
+    to the app as a router.
+
+    The health check endpoint is available via a GET request to the `/health`
+    route and returns a 200 response if the server is active.
+
+    For example, if the FastAPI app is running locally on
+    `http://localhost:8000`, the health check endpoint can be accessed via
+    a GET request query to `http://localhost:8000/health`.
+
+    Example usage in FastAPI app:
+        ```python hl_lines="2 5"
```

Comment:
Is this `hl_lines` syntax related to MKdocs? Im curious how I can preview changes like this on PRs

Reply:
You can run `make view-docs` locally and then navigate to the page documents this file.
In this case it's to highlight line 2 and 5.